### ISSUE TITLE
Impossible to connect to Keybase services

### ISSUE THREAD

**Issue by Japoneris** [keybase/client#26123]
On smartphone and laptop, got:

ERROR CODE 1601 - API network error (see image bellow)

![Capture d’écran du 2024-01-05 17-07-52](https://github.com/keybase/client/assets/9401619/e588eb63-0b45-4f6e-b39c-ac4394537aee)

Impossible to see contact / send message / access private folders.

Thanks

**Comment by soltrinox** [comment]
Yep same here. Looks like the merge with ZOOM has not been good for the network.

**Comment by foss-** [comment]
Re-install from https://keybase.io/ should resolve this certificate problem.

**Comment by lucymhdavies** [comment]
<img width="648" alt="image" src="https://github.com/keybase/client/assets/1116529/d956df49-01bf-45f1-827f-49f5f8d768b3">

re-installed from https://keybase.io/ and yep, that solves the issue
